app/html/filter.js

```diff
@@ -23,15 +25,20 @@ exports.create = function (api) {
 
     const myId = api.keys.sync.id()
     const peopleIFollow = api.contact.obs.following(myId)
-    const onlyPeopleIFollow = Value(false)
+    const onlyPeopleIFollow = Value(api.settings.sync.get('filter.onlyPeopleIFollow') || false)
     const onlyAuthor = Value()
 
-    const showPost = Value(true)
-    const showAbout = Value(true)
-    const showVote = Value(false)
-    const showContact = Value(false)
-    const showChannel = Value(false)
-    const showPub = Value(false)
+    const showPost = Value(api.settings.sync.get('filter.showPost') || true)
+    const showAbout = Value(api.settings.sync.get('filter.showAbout') || true)
+    const showVote = Value(api.settings.sync.get('filter.showVote') || false)
+    const showContact = Value(api.settings.sync.get('filter.showContact') || false)
+    const showChannel = Value(api.settings.sync.get('filter.showChannel') || false)
+    const showPub = Value(api.settings.sync.get('filter.showPub') || false)
+    const showChess = Value(api.settings.sync.get('filter.showChess') || true)
```

Comment:
couple of things here : 
- get has a built in fallback. Go read lodash/get. Basically this could be : api.settings.sync.get('filter.showChess', true)
- I think I want to set a default for the app settings .. might put that in `app.sync.settingsDefaults`

Reply:
- `api.settings.obs.get` actually get your an observeable value